### DISCUSSION TITLE
Multiple timeseries

### DISCUSSION THREAD

**Discussion by sukirboy** [teamalgoritma/community#161] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88123718/175920122-7e00d810-5070-4cff-a9f2-f962968e1f27.png)

mau konfirmasi dan tanya kak..

itu tailnya angka 24 diambil dari frekuensi jam dalam 1 hari, dan 7 itu diambil dari frekuensi hari didalam minggu.. benar gak kak?

trs klo 4 itu drmn yah kak?

**Comment by onlyphantom** [comment, marked as answer]
Seems reasonable to guess that it's from 4 weeks in a month. 

Jadi:
24 hours a day, 7 days a week, 4 weeks a month.

Not an official answer, but seems a reasonable guess to me.
